pyshelf/search/metadata.py

```diff
@@ -1,11 +1,15 @@
-from elasticsearch_dsl import String, Nested, Boolean, DocType
+from elasticsearch_dsl import String, Nested, Boolean, DocType, tokenizer, analyzer
+
+
+# Required for case sensitivity
+metadata_analyzer = analyzer("metadata_analyzer", tokenizer=tokenizer("keyword"))
```

Comment:
erm, I think we may have had a miscommunication about it.  The value doesn't need to match case insensitivity, I meant the sort_type/sort_flag should be case insensitive.  I don't think we would want this (unless we had a special character sent to specify it or something).